web/source/text/defaultOutput.ts

```diff
@@ -140,8 +198,8 @@ namespace com.keyman.text {
       if (((0x0 <= codePoint) && (codePoint <= 0x1F)) || ((0x80 <= codePoint) && (codePoint <= 0x9F))) {
         // Code points [U_0000 - U_001F] and [U_0080 - U_009F] refer to Unicode C0 and C1 control codes.
         // Check the codePoint number and do not allow output of these codes via U_xxxxxx shortcuts.
-        if(!quiet) {
-          console.log("Suppressing Unicode control code: U_00" + codePoint.toString(16));
+        if(ruleBehavior) {
+          ruleBehavior.errorLog = ("Suppressing Unicode control code: U_00" + codePoint.toString(16));
```

Comment:
Does this imply only a single error can be raised in a given process?

Reply:
It does imply this, yes.  I could set things up to log multiple entries, but I felt that needlessly complex for the current implementation; only one may currently be applied for any given path through the existing code base.